client/templates/other/welcome.html

```diff
@@ -0,0 +1,71 @@
+<template name="welcome">
+  <div>
+    <section class="conatainer-fluid section-header">
+      <h1>Welcome Guide</h1>
+      <a href="{{pathFor 'all study groups'}}" class="btn btn-link">
```

Comment:
50/50 here but will leave it to you @julianjohannesen - 

Should this link back to `/hangouts` as that's the page that you land in after a user signs in? If so, we should change the text to "Start or Join a Hangout"

Likewise, we should change the text to "Start or Join a Study Group" to encourage more actionable feedback to the user.

Reply:
Sorry about the delayed reply. From what Linda was telling me, it sounded like there was a longer conversation about where it should link to. Not sure what makes the most sense. I like the "start or join" language.

Reply:
I like the "start or join" language too! Will merge this in for now, and we can make the language tweaks in a new issue/PR.